cluster-template.yaml

```diff
@@ -0,0 +1,1540 @@
+#  # (1) allow access from controller nodes to services running on an existing infrastructure
+#  # (2) allow ssh accesses from bastions when `sshAccessAllowedSourceCIDRs` are explicitly set to an empty array
+  securityGroupIds:
+    # allow SSH from internal VPC machines. Group name "Internal SSH security group"
+    # todo [customize] per account & region
+    - $VPC_INTERNAL_SSH_SG
+#
+#  # Auto Scaling Group definition for controllers. If only `controllerCount` is specified, min and max will be the set to that value and `rollingUpdateMinInstancesInService` will be one less.
+#  autoScalingGroup:
+#    minSize: 1
+#    maxSize: 2
+#    rollingUpdateMinInstancesInService: 1
+#  # If you specify managedIamRoleName the role created for controller nodes will not suffix the random id at the end
+#  # Role will be created with "Ref": {"AWS::StackName"}-{"AWS::Region"}-yourManagedRole
+#  # to follow the recommendation in AWS documentation  http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html
+#  # This is also intended to be used in combination with .experimental.kube2IamSupport. See #297 for more information.
+#  #
+#  # ATTENTION: Consider limiting number of characters in clusterName and iam.role.name to avoid the resulting IAM
+#  # role name's length from exceeding the AWS limit: 64
+#  # See https://github.com/kubernetes-incubator/kube-aws/issues/347
+#  # if you omit this block kube-aws would create an IAM Role and a customer managed policy with a random name.
+  iam:
+#    role:
+#       # This will create a named IAM Role managed by kube-aws with the name {AWS::Region}-$managedIamRoleName.
+#       # It will have attached a customer Managed Policy that you can modify afterwards if you need more permissions for your cluster.
+#       # Be careful with the Statements you modify because an update could overwrite your own.
+#       # the Statements included in the ManagedPolicy are the minimun ones required for the Controllers to run.
+#       # name: "yourManagedRole"
+#       # if you set managedPolicies here it will be attached in addition to the created managedPolicy in kube-aws for the cluster.
+#       # CAUTION: if you attach a more restrictive policy in some resources (i.e ec2:* Deny) you can make kube-aws fail.
+#       # managedPolicies:
+#       # -  arn: "arn:aws:iam::aws:policy/AdministratorAccess"
+#       # -  arn: "arn:aws:iam::YOURACCOUNTID:policy/YOURPOLICYNAME"
+
+#       # if you set an InstanceProfile kube-aws will NOT create any IAM Role and will use the configured instanceProfile.
+#       # CAUTION: you must ensure that the IAM Role linked to the listed InstanceProfile has enough permissions to ensure kube-aws to run.
+#       # if you dont know which permissions are required is recommended to create a cluster with a managed role.
+    instanceProfile:
+      # todo [customize] per account & region
+      arn: "$CONTROLLER_IAM_ROLE"
```

Comment:
Can we lose the quotes to keep it consistent with the way the arn is substituted for the workers and the  etcd instances?